bramkouwen/Kouwen_Abram_AV_ResearchProject-
# adding public folder

## What does the PR do?
* It adds the public folder 

## How to check this PR?
* Pull down the branch
* See added files
